openssh/files/sshd_config

```diff
@@ -69,7 +68,7 @@
 
 # Authentication:
 {{ option_default_uncommented('LoginGraceTime', 120) }}
-{{ option_default_uncommented('PermitRootLogin', 'no') }}
+{{ option_default_uncommented('PermitRootLogin', 'yes') }}
```

Comment:
What was the reason for this change? Allowing someone to log in as root is a big red flag.